easybuild/easyblocks/b/boost.py

```diff
@@ -311,21 +337,23 @@ def sanity_check_step(self):
         else:
             custom_paths['files'].append(os.path.join('lib', 'libboost_system.%s' % shlib_ext))
 
-            lib_mt_suffix = '-mt'
-            # MT libraries gained an extra suffix from v1.69.0 onwards
-            if LooseVersion(self.version) >= LooseVersion("1.69.0"):
-                if get_cpu_architecture() == AARCH64:
-                    lib_mt_suffix += '-a64'
-                elif get_cpu_architecture() == POWER:
-                    lib_mt_suffix += '-p64'
-                else:
-                    lib_mt_suffix += '-x64'
+            if self.cfg['tagged_layout']:
+                lib_mt_suffix = '-mt'
```

Comment:
You need to set lib_mt_suffix to empty when not using the tagged layout.

Reply:
No because it's only used when tagged_layout is set.

Reply:
Ah true, got it :+1: